Source/FFmpegMediaSource.cpp

```diff
@@ -741,11 +741,27 @@ namespace winrt::FFmpegInteropX::implementation
             }
 
             //inject custom properties
-            if (config->AutoCorrectAnsiSubtitles() && config->IsExternalSubtitleParser && streamByteOrderMark != ByteOrderMark::UTF8)
+            if (config->AutoCorrectAnsiSubtitles() && config->IsExternalSubtitleParser &&
```

Comment:
Shouldn't the option be named  `AutoDetectSubtitleEncoding` or similar?

Reply:
Good point, we should probably rename that for 2.0 release.

Reply:
Naming improved